clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -1996,16 +1996,21 @@ struct CallObjCArcUse final : EHScopeStack::Cleanup {
 
 Value *CodeGenFunction::EmitCheckedArgForBuiltin(const Expr *E,
```

Comment:
TO my taste, I'd rather forked EmitCheckedArgForBuiltin into a version for Assume
very little code is shared with Zero

Reply:
Done